lib/elixir/src/elixir_clauses.erl

```diff
@@ -184,11 +184,16 @@ expand_with_else(Meta, Opts, E, HasMatch) ->
   RaiseError = fun(Key) ->
     form_error(Meta, ?key(E, file), ?MODULE, {duplicated_clauses, 'try', Key})
   end,
+  CatchBeforeRescueWarn = fun() ->
```

Comment:
It doesn't seem we need the fun, we can just inline this code in check_catch_after_rescue. What do you think?

Reply:
ok